src/pretix/control/forms/item.py

```diff
@@ -172,6 +218,17 @@ def save(self, *args, **kwargs):
 
         return instance
 
+    def clean(self):
+        cleaned_data = super().clean()
+
+        if self.cleaned_data.get('quota_option') == '2':
+            if not self.cleaned_data.get('quota_add_new_name'):
+                raise forms.ValidationError(
+                    {'quota_add_new_name': ['Quota name is required.']}
```

Comment:
Same here, should be translatable. Also, shouldn't the size be required in this case as well?

Also, if an existing quota is used, that field should be required.